CHANGELOG.md

```
@@ -1,14 +1,130 @@
+
+### ZeroMQ broker: run AiiDA without external services
+
+AiiDA requires a message broker to communicate with running processes and the daemon.
+Until now, the only option was RabbitMQ, an external service that has to be installed and managed separately, which complicates installation, especially on machines without administrator rights (e.g., HPC login nodes).
+
+This release adds a built-in **ZeroMQ broker** as a new broker plugin (`core.zeromq`, next to the existing `core.rabbitmq`).
+It requires no external services: the broker is started and supervised automatically together with the daemon.
+This means full process control—submitting to the daemon with `submit()`, `verdi process play/pause/kill`, `verdi daemon start/stop`—now works out of the box with nothing but `pip install aiida-core`:
+
+```console
+$ pip install aiida-core
```

markdownlint-cli2 (0.23.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)